recipe/meta.yaml

```
@@ -40,7 +40,7 @@ requirements:
     - python
     - pip
     - setuptools
-    - cython >=0.24.0
+    - cython >=0.29.22
```

Same nitpick as I made in another PR:
```suggestion
    - cython >=0.29.22,<3
```